provider/src/main/java/com/synopsys/integration/alert/provider/blackduck/task/accumulator/BlackDuckAccumulator.java

```diff
@@ -129,15 +132,22 @@ private void storeNotifications(List<NotificationView> notifications) {
 
     private List<AlertNotificationModel> convertToAlertNotificationModels(List<NotificationView> notifications) {
         return notifications
-            .stream()
-            .sorted(Comparator.comparing(NotificationView::getCreatedAt))
-            .map(this::convertToAlertNotificationModel)
-            .collect(Collectors.toList());
+                   .stream()
+                   .sorted(Comparator.comparing(NotificationView::getCreatedAt))
+                   .map(this::convertToAlertNotificationModel)
+                   .collect(Collectors.toList());
     }
 
     private void write(List<AlertNotificationModel> contentList) {
         logger.info("Writing {} notifications...", contentList.size());
-        notificationAccessor.saveAllNotifications(contentList);
+        List<AlertNotificationModel> savedNotifications = notificationAccessor.saveAllNotifications(contentList);
+        if (logger.isDebugEnabled()) {
+            List<Long> notificationIds = savedNotifications.stream()
+                                             .map(AlertNotificationModel::getId)
+                                             .collect(Collectors.toList());
+            String joinedIds = StringUtils.join(notificationIds, ", ");
+            notificationLogger.debug("Saving notifications: {}", joinedIds);
```

Comment:
You might want to change this to `Saved Notifications` just to be clear it's what was written to the database.